lighthouse-core/gather/driver.js

```diff
@@ -460,27 +520,28 @@ class Driver {
    * See https://github.com/GoogleChrome/lighthouse/issues/627 for more.
```

Comment:
this method description need an update. Can you flesh it out a bit so we won't have to look back up the PR when trying to read the code?